setup.md

```diff
@@ -9,31 +9,37 @@ You need to install OpenRefine and download a data file to follow this lesson.
+* In OpenRefine versions 3.4+ the options "Windows kit with embedded Java" and "Mac kit" include Java as part of the package. You **do not** need to install Java if you use one of these kits. This is the preferred method on Windows and Mac systems.
+* On Windows, if you use the traditional "Windows kit" without embedded Java, you will need a "Java Runtime Environment" (JRE) on your system. If you do not already have JRE or JDK installed, you can visit [Adopt OpenJDK](https://adoptopenjdk.net/) or [Oracle Java](https://java.com/en/download/) to download an installer package. Please note that [Oracle significantly changed their license terms in 2019](https://www.oracle.com/java/technologies/javase/jdk-faqs.html) limiting it to "personal use" with out a paid license. If you use OpenRefine at work or in research, OpenJDK is preferred.
+* On Linux a "Java Runtime Environment" (JRE) will be required to run OpenRefine. If you do not already have JRE or JDK installed on your system, most distribution repositories will contain OpenJRE / OpenJDK packages. Install the default version available from your distribution. For example, on Ubuntu/Debian: `sudo apt install default-jre`.
+* OpenRefine does not support Internet Explorer. Please use [Firefox](https://www.mozilla.org/firefox/new/), [Chrome](https://www.google.com/chrome/) or [Safari](https://www.apple.com/safari/) instead.
 
 ### Downloading the data
 
 You can download [doaj-article-sample.csv](https://github.com/LibraryCarpentry/lc-open-refine/raw/gh-pages/data/doaj-article-sample.csv), which is a csv file that will open in a new browser tab. Be sure to right click or control click in order to save the file (NOTE: In Safari, right click and select **download linked file**; in Chrome and Firefox, right click and select **save link as...**). Make a note of the location (i.e. the folder, your desktop) to which you save the file.
 
 ### Getting help
 
-If you encounter problems installing or running OpenRefine, a good source of support is [the OpenRefine mailing list and forum](https://groups.google.com/g/openrefine).
-
-If you are installing OpenRefine on Windows, you may want to check the forum for ['Windows' related threads](https://groups.google.com/forum/?fromgroups#!searchin/openrefine/windows%7Csort:date) or specific threads like [Installing OpenRefine on Windows 7](https://groups.google.com/forum/?fromgroups#!searchin/openrefine/64-bit%7Csort:date/openrefine/vUzqJqJ-sAA/Tb2Om9wvaqgJ).
+If you encounter problems installing or running OpenRefine, good sources of support are the [OpenRefine Google Group](https://groups.google.com/g/openrefine) or [Stack Overflow OpenRefine tag](https://stackoverflow.com/questions/tagged/openrefine).
+Include your operating system when searching to find the most relevant answers for your issue.
```

Comment:
I'm not sure about this sentence in place of the previous links to the forum with search terms setup? I appreciate that the Windows 7 link is now outdated and needs removing, but maybe we could use links which point to searches for Windows / macOS / Linux?

Don't feel strongly about this but would appreciate input from others

Reply:
I agree that providing links to specific help is...helpful, but there is the problem with needing to update. You think search links would be a good solution, @ostephens ?

Reply:
Would the search strings be like 
["install on windows"](https://groups.google.com/g/openrefine/search?q=install%20on%20windows)
["install on mac"](https://groups.google.com/g/openrefine/search?q=install%20on%20mac)
["install on linux"](https://groups.google.com/g/openrefine/search?q=install%20on%20linux)
?

I am not sure how helpful the search links are. People will likely need to concoct a more detailed query to find something relevant anyway, so I guess the links are a starting point?

Reply:
@ostephens I just pushed adding the search links to the paragraph, let me know if it makes sense.